src/write/index.internals.js

```diff
@@ -32,34 +32,34 @@ module.exports = function(destPath, options) {
   function mapSources(file) {
     var debug = rootDebug.spawn('mapSources');
 
-    //NOTE: make sure source mapping happens after content has been loaded
+    // NOTE: make sure source mapping happens after content has been loaded
     if (options.mapSources && typeof options.mapSources === 'function') {
       debug(function() { return '**Option is deprecated, update to use sourcemap.mapSources stream**'; });
       debug(function() { return 'function'; });
 
-      file.sourceMap.sources = file.sourceMap.sources.map(function (filePath) {
+      file.sourceMap.sources = file.sourceMap.sources.map(function(filePath) {
         return options.mapSources(filePath, file);
       });
       return;
     }
 
-    debug(function() { return "file.path: " + file.path; });
-    debug(function() { return "file.cwd: " + file.cwd; });
-    debug(function() { return "file.base: " + file.base; });
+    debug(function() { return 'file.path: ' + file.path; });
+    debug(function() { return 'file.cwd: ' + file.cwd; });
+    debug(function() { return 'file.base: ' + file.base; });
 
     file.sourceMap.sources = file.sourceMap.sources.map(function(filePath) {
       // keep the references files like ../node_modules within the sourceRoot
-      debug(function() { return "filePath: " + filePath; });
+      debug(function() { return 'filePath: ' + filePath; });
 
-      if (options.mapSourcesAbsolute === true){
+      if (options.mapSourcesAbsolute === true) {
         debug(function() { return 'mapSourcesAbsolute'; });
 
-        if (!file.dirname){
+        if (!file.dirname) {
           debug(function() { return '!file.dirname'; });
           filePath = path.join(file.base, filePath).replace(file.cwd, '');
```

Comment:
Before this PR the tests hit this line, now it does not as `file.dirname` is never false.

Reply:
I believe that's due to the vinyl or gulp 4 update. I think it's fine to leave for older vinyl or gulp.